LayoutTests/fast/canvas/canvas-filter-font-relative-unit.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<body>
+    <canvas id="canvas"></canvas>
+    <p>This test passes if it does not crash</p>
+
+    <script>
+        if (window.testRunner) {
+            testRunner.waitUntilDone();
+            testRunner.dumpAsText();
+
+            // Let the test run for a bit so WebKitTestRunner could
+            // catch the GPU process crash.
+            setInterval(() => testRunner.notifyDone(), 1000);
```

Comment:
What does the comment `Let the test run for a bit` mean? I think this `setInterval()` and `waitUntilDone()` are not needed.

Reply:
This is removed.